crates/rpc/rpc-eth-api/src/helpers/state.rs

```diff
@@ -127,6 +128,33 @@ pub trait EthState: LoadState + SpawnBlocking {
             .await
         })
     }
+
+    /// Returns the account at the given address for the provided block identifier.
+    fn get_account(
+        &self,
+        address: Address,
+        block_id: BlockId,
+    ) -> impl Future<Output = Result<Account, Self::Error>> + Send {
+        self.spawn_blocking_io(move |this| {
+            let state = this.state_at_block_id(block_id)?;
+
+            let account = state
+                .basic_account(address)
+                .map_err(Self::Error::from_eth_err)?
+                .unwrap_or_default();
+            let balance = account.balance;
+            let nonce = account.nonce;
+            let code_hash = account.bytecode_hash.unwrap_or_default();
```

Comment:
actually, this will be 0x000 whereas it should be `KECCAK_EMPTY`

Reply:
of course sorry for missing this